static/js/services/markdown.js

```diff
@@ -0,0 +1,60 @@
+var inboxServices = angular.module('inboxServices');
```

Comment:
Is there not a 3rd party library we could use here instead of implementing our own?